getting-started/python/index.md

```diff
@@ -91,75 +89,71 @@ or the [openEO Hub](https://hub.openeo.org/) for collection listings of other ba
 Processes in openEO are operations that can be applied on (EO) data
 (e.g. calculate the mean of an array, or mask out observations outside a given polygon).
 The output of one process can be used as the input of another process,
-and by doing so, multiple processes can be connected that way in a larger "process graph":
-a new (user-defined) processes that implements a certain algorithm.
+and by doing so, multiple processes can be connected that way in a larger "process graph"
+that implements a certain algorithm.
 
 ::: tip Note
 Check [the openEO glossary](https://openeo.org/documentation/1.0/glossary.html#processes)
 for more details on pre-defined, user-defined processes and process graphs.
 :::
 
 
-Let's list the (pre-defined) processes available on the back-end
+Let's list the available pre-defined processes 
 with [`list_processes`](https://open-eo.github.io/openeo-python-client/api.html#openeo.rest.connection.Connection.list_processes):
 
-```python
-print(connection.list_processes())
 ```
-
-which returns a list of dictionaries describing the process (including expected arguments and return type), e.g.: 
-
-```
-[{'id': 'absolute', 'summary': 'Absolute value', 'description': 'Computes the absolute value of a real number `x`, which is th...', 
- {'id': 'mean', 'summary': 'Arithmetic mean(average)', ...}
- ...]
+>>> print(connection.list_processes())
```

Comment:
In general regarding the print statements: Adding print here is mainly for non-Jupyter envs, right? Because adding print in the examples suppresses the nice UI components in Jupyter, which we should maybe mention or so?

Reply:
yes indeed, that is a general python documentation issue: do explicit print or let implicit `__repr__`/`__repr_html__` do it's job.

I think I here added the `print` to make sure that the output is this JSON-like dump as shown. I also added the `>>>` prompt (suggesting a Python/IPython shell) to have both the statement and it's output in the same code block.

but indeed, mentioning the Jupyter integration here and in other places is a good idea